site/profile/files/html/css/style.css

```diff
@@ -185,3 +185,27 @@ select:focus {
 .btn-inverse {
     background-color: #222;
 }
+
+/* customizations */
+.get-started {
+    display: block;
+    margin: 2em auto 0;
+}
+
+#faq li {
+  cursor: pointer;
+}
+#faq li h4 {
+  margin: 0;
+}
+#faq li .answer > :first-child {
+  margin-top: 0;
+}
+
+article.tutorial img {
+  float: right;
+}
+article.tutorial div.break {
```

Comment:
Why not just use an <hr>?

Reply:
You can change it if you want, but you'd still have to style it with a `clear: both` so it's not like it changes much.